policyengine_us/parameters/gov/states/ny/tax/income/credits/non_refundable.yaml

```diff
@@ -4,4 +4,4 @@ values:
     - ny_household_credit
```

Comment:
change description:
"New York provides the following non-refundable tax credits."